packages/ui-components/src/components/breadcrumb-item/breadcrumb-item.tsx

```diff
@@ -0,0 +1,57 @@
+import { Component, Host, h, Prop, EventEmitter, Event } from '@stencil/core';
+import { throttle } from 'lodash-es';
+import { IBreadcrumbItem, EBreadcrumbItemTarget } from './breadcrumb-item.types';
+
+@Component({
+	tag: 'kv-breadcrumb-item',
+	styleUrl: 'breadcrumb-item.scss',
+	shadow: true,
+})
+export class KvBreadcrumbItem {
+	/** (required) The text to display on the breadcrumb */
+	@Prop({ reflect: true }) label!: string;
+	/** (optional) The breadcrumb's link */
+	@Prop({ reflect: true }) href?: string;
+	/** (optional) The target of the link (only used if href is provided) */
+	@Prop({ reflect: true }) target?: EBreadcrumbItemTarget;
+	/** (optional) Sets this breadcrumb styling to be the active one (usually the last one) */
+	@Prop({ reflect: true }) active?: boolean;
+	/** (optional) The separator to use */
+	@Prop() separator?: string;
```

Comment:
I don't think that this component should have context about a separator.

This should be only on the `breadcrumb.tsx` component since it is the one that wraps all the `breadcrumb-item`s.

Also, you are assuming here that if the breadcrumb is "active" is the last one and you should render the separator, which in some cases could not be true.